src/public/scripts/attractions-facilities-toggle-slides.js

```diff
@@ -0,0 +1,103 @@
+let article_slide_index = 1;
+
+//js for article 1
+article_show_slides_1(article_slide_index);
```

Comment:
You should probably make this more generic, just one function maybe called `articleShowSlides`  which receives the index and you decide on the fly by interpolating the string, eg. `article-<index>-mySlides`